image.yaml

```diff
@@ -151,5 +151,5 @@ run:
           - "/opt/amq/bin/launch.sh"
 osbs:
       repository:
-            name: jboss-amq-6-docker
+            name: containers/jboss-amq-6
             branch: ce-1.3-openshift-amq-6.3-jdk-8-rhel-7
```

Comment:
Branch is different too: `jb-amq-6.3-openshift-rhel-7`. The suggestion is that the branch in the image image descriptor is pointing to the "main" target branch, but the specific one (`jb-amq-6.3-openshift-cp-rhel-7`) is added to an override file.